lib/hammer_cli_foreman/compute_resource/register_compute_resources.rb

```diff
@@ -1,11 +1,17 @@
 module HammerCLIForeman
   @compute_resources = {}
+  @interfaces_attrs_name_list ={}
```

Comment:
What benefit it has to store this duplicate information in here? I'd prefer to not add to top lavel namespace more things then it is necessary.
```ruby
HammerCLIForeman.compute_resources['ovirt'].interfaces_attrs_name
```
feels readable enough.